internal/testutil/error.go

```diff
@@ -0,0 +1,11 @@
+package testutil
+
+import (
+	"runtime"
+
+	"github.com/samber/lo"
+)
+
+var ErrNotExist string = lo.Ternary(runtime.GOOS == "windows",
+	"The system cannot find the file specified.",
+	"no such file or directory")
```

Comment:
Why the messaging is different based on the OS?

Reply:
I guess the messages are from OS.